backend/sozisel/test/sozisel_web/schema/events/event_subscriptions_test.exs

```diff
@@ -251,6 +269,59 @@ defmodule SoziselWeb.Schema.Events.EventSubscriptionsTest do
              } = receive_subscription(sub)
     end
 
+    test "broadcast whiteboard event result back to presenter", ctx do
+      event = insert(:whiteboard_event, session_template_id: ctx.session_template.id)
+      launched_event = insert(:launched_event, event_id: event.id, session_id: ctx.session.id)
+
+      variables = %{
+        sessionId: ctx.session.id
+      }
+
+      sub = run_subscription(ctx.user_socket, @presenter_event_result_submitted, variables)
+
+      File.mkdir("/tmp/images")
+      File.copy!("test/assets/test_image.png", "/tmp/images/test_image.png")
+
+      upload = %Plug.Upload{
+        content_type: "image/png",
+        path: "/tmp/images/test_image.png",
+        filename: "whiteboard_image.png"
+      }
+
+      variables = %{
+        input: %{
+          launched_event_id: launched_event.id,
+          image: "image",
+          path: "some path",
+          text: "some text",
+          used_time: 145
+        },
+        token: ctx.participant.token
+      }
+
+      assert %{"data" => %{"submitWhiteboardResult" => %{"id" => launched_event_id}}} =
+               ctx.conn
+               |> post("/api/",
+                 query: @submit_whiteboard_result,
+                 variables: variables,
+                 image: upload
+               )
+               |> json_response(200)
```

Comment:
can you check that the image got persisted on the disk under a right name? Because generated name for sure won't be `test_image.png`.

Reply:
done